src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java

```diff
@@ -22,15 +22,18 @@ public Map<String, Object> mapCaseDataToParams(CaseData caseData) {
         params.put("defaultRespondTime", "4pm");
         params.put("defendantName", caseData.getRespondent1().getPartyName());
 
-        params.put(
-            "claimFee",
-            "£" + MonetaryConversions.penniesToPounds(caseData.getClaimFee().getCalculatedAmountInPence())
-                .stripTrailingZeros().toPlainString()
-        );
+        if (nonNull(caseData.getClaimFee())) {
+            params.put(
+                "claimFee",
+                "£" + MonetaryConversions.penniesToPounds(caseData.getClaimFee().getCalculatedAmountInPence())
+                    .stripTrailingZeros().toPlainString()
+            );
+        }
 
         if (nonNull(caseData.getRespondent1ResponseDeadline())) {
             params.put("responseDeadline", DateUtils.formatDate(caseData.getRespondent1ResponseDeadline()));
         }
+        params.put("typeOfFee", "claim fee");
```

Comment:
make the name specific like `claimFeeType` as there might be another entry for `hearingFeeType`